src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -137,6 +137,10 @@ module.exports = class LevelBlock {
       this.isRedstoneBattery = blockType === 'pressurePlateUp' ? false : true;
     }
 
+    if (blockType === "glowstone") {
+      this.isEntity = true;
```

Comment:
why is glowstone an entity?

Reply:
Because entity doesn't mean entity. Blocks tagged as entity just mean "do not draw shadows / AO". Josh and I have spoken about how that's a really dumb name for how it's handled / what it effects. 😞

Reply:
Can we update the name to reflect the new behavior?  Thanks!